docs/rtd/source/Commands.rst

```diff
@@ -0,0 +1,107 @@
+Commands
```

Comment:
Can we regenerate this from the man pages or from the command, rather then making someone edit for every new command.

Reply:
this is just to get the ball rolling ...